CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 * **[Fix]** Fix crash involving SDK's `ms_viewWillAppear` method.
 
+### App Center Crashes
+
+* **[Fix]** Fix `MSCrashesDelegate` callback methods not being called on the main thread.
```

Comment:
@achocron We want to mention this as a behavior change, right?

Reply:
```suggestion
* **[Behavior Change]** Fix `MSCrashesDelegate` callback methods not being called on the main thread.
```